src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/OpenAppManifestValueProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    [ExportInterceptingPropertyValueProvider("CreateOrOpenAppManifest", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
```

Comment:
(random thought) When writing up the docs on interception, I realised we can have one `NoOpInterceptingPropertyValueProvider` for multiple properties (by passing an array of strings for the first argument). Maybe we look to do that in future, to reduce the number of source files and types we ship.